addons/api/tests/unit/serializers/target-test.js

```diff
@@ -145,6 +148,78 @@ module('Unit | Serializer | target', function (hooks) {
       session_max_seconds: 28800,
       session_connection_limit: null,
       worker_filter: 'worker',
+      egress_worker_filter: null,
+      ingress_worker_filter: null,
+      attributes: {
+        default_port: 1234,
+      },
+      address: null,
+    });
+  });
+
+  test('it serializes the egress_worker_filter attribute if present', function (assert) {
+    assert.expect(1);
+    const store = this.owner.lookup('service:store');
+    const serializer = store.serializerFor('target');
+    const record = store.createRecord('target', {
+      name: 'User',
+      description: 'Description',
+      version: 1,
+      type: TYPE_TARGET_SSH,
+      egress_worker_filter: 'egress worker',
+      default_port: 1234,
+      scope: {
+        scope_id: 'org_1',
+      },
+    });
+    const snapshot = record._createSnapshot();
+    const serializedRecord = serializer.serialize(snapshot);
+    assert.deepEqual(serializedRecord, {
+      name: 'User',
+      description: 'Description',
+      version: 1,
+      type: TYPE_TARGET_SSH,
+      scope_id: 'org_1',
+      session_max_seconds: 28800,
+      session_connection_limit: null,
+      worker_filter: null,
+      egress_worker_filter: 'egress worker',
+      ingress_worker_filter: null,
+      attributes: {
+        default_port: 1234,
+      },
+      address: null,
```

Comment:
do we have an address field?

Reply:
Yes. @ZedLi just added that very recently for work that he is doing.